New package: ITensorNumericalAnalysis v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: ITensorNumericalAnalysis
- Repository: https://github.com/JoeyT1994/ITensorNumericalAnalysis.jl
- Created by: @JoeyT1994
- Version: v0.1.0
- Commit: bbdec04d7f152a46e2bbada896220ddc95d4c703
- Reviewed by: @JoeyT1994
- Reference: https://github.com/JoeyT1994/ITensorNumericalAnalysis.jl/commit/bbdec04d7f152a46e2bbada896220ddc95d4c703#commitcomment-141360600
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6ebb9c1e941b0fb79951737dffb8666df2e6ea0b1b88bc9cc13fe156b2b0c765b254f5d71fcd1524b50ee7e452db2ce2f9787dbb2f737629aa070303d5e3ac0d24afae244aa7c963671d96fdebe46485e55fbe3382d2da23c90d88f319b5e533868879bc3aa5cbbe91ef19f685c2948329a914f6b3bfaa54579773c67668abcb3b6f598c4352bdf0677212a679148e2cf9becf675710d0365c8c7e07dc9275dfa93ece2c2e07f74bc1332528b2a53501dd4a85921b2485d5931a665bb672de109 -->

goerz commented:
Thank you for submitting your package! Could you add some documentation before registering, to give an indication for how the Julia API specifically can be used?

JuliaRegistrator committed:
New package: ITensorNumericalAnalysis v0.1.0

UUID: 666f501e-685d-4e36-ab44-ece85df6022b
Repo: https://github.com/JoeyT1994/ITensorNumericalAnalysis.jl.git
Tree: 4622b92aab4b5f39bc47a19d1b18d8703976b1c0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
